packages/web-client/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Logo from '../../assets/icons/skillfuze-logo.svg';
+
+interface Props {
+  controls?: React.ReactNode;
+}
+
+const Navbar: React.FC<Props> = (props: Props) => (
+  <nav className="flex py-3 px-5">
+    <div className="flex-grow">
+      <Logo style={{ width: '7.5rem' }} />
```

Comment:
I think we should use tailwind, I was told to use approximate values.

Reply:
Tailwind gives us a 6rem, and 8rem values. to exactly match the size we want, a custom width had to be used.

https://tailwindcss.com/docs/width